<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountStatsDialog</class>
 <widget class="QDialog" name="AccountStatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelAccountImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Image</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableStats">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>30</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelAccountName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Account Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeInterval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Time Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccountCurrency">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Account Currency</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccountLanguage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="text">
     <string>Account Language</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLanguage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="text">
     <string>Language:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrency">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="text">
     <string>Currency:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelAdditionalInformation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Additional information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
